packages/prefresh/transform/src/lib.rs

```diff
@@ -0,0 +1,289 @@
+use std::{
+    collections::{HashMap, HashSet},
+    hash::{DefaultHasher, Hasher},
+};
+
+use serde::Deserialize;
+use swc_atoms::Atom;
+use swc_common::DUMMY_SP;
+use swc_core::quote;
+use swc_ecma_ast::{
+    AssignExpr, Callee, ComputedPropName, Expr, ExprOrSpread, Function, Id, Ident, ImportDecl,
+    ImportSpecifier, Lit, MemberExpr, ModuleExportName, ObjectPatProp, Tpl, TplElement,
+    VarDeclarator,
+};
+use swc_ecma_visit::{as_folder, Fold, VisitMut, VisitMutWith};
+
+pub fn swc_prefresh(config: PrefreshPluginConfig, file_hash: String) -> impl Fold {
+    as_folder(PrefreshPlugin::new(config, file_hash))
+}
+
+#[derive(Debug, Clone, Deserialize, Default)]
+#[serde(rename_all = "camelCase")]
+pub struct PrefreshPluginConfig {
+    pub library: Option<Vec<String>>,
+}
+
```

Comment:
```suggestion
fn default_library() -> Vec<String> {
    vec!["preact".into(), "react".into()]
}
```

In this way we can avoid repeating allocating for each import statements